.github/workflows/ci_linux_arm64_clang.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - Linux arm64 clang
+
+on:
+  schedule:
+    # Weekday mornings at 09:15 UTC = 01:15 PST (UTC - 8).
+    - cron: "15 9 * * 1-5"
+  workflow_dispatch:
```

Comment:
@banach-space my thinking here is that you could go in to this file and change the triggers to fit your resourcing

So if you have a single runner (4+ cores) it could run nightly. If you have a single powerful runner (16+ cores), it could run on `push`. If you have multiple powerful runners (32+ cores), it could run on `pull_request`.

If runners go offline for whatever reason (cloud pool like GCP needs maintenance, self-hosted lab needs a local reboot, etc.), we can disable the job from the GitHub UI

---

I'm planning to sort of hack and slash through CI configuration this week and then follow up with some docs (maybe updating https://iree.dev/developers/general/contributing/#continuous-integration-ci) once the details have settled.

Reply:
Thanks, this stuff is fairly straightforward, thanks for setting this up 🙏🏻